### PR TITLE
Epoch Speed Improvement

### LINKED ISSUE #834: AutoKeras 1.0 much slower than 0.4 on Google Colab
When I try to run a simple MNIST example on Google Colab with GPU with Autokeras 0.4 it runs very fast (1 epoch of the first model takes < 2 s) but with 1.0 it runs much slower (1 epoch of the first model takes > 80 s). When I disable the GPU 0.4 runs as slow as 1.0 which suggests that 1.0 isn’t using the GPU. How can I make Autokeras 1.0 run as fast as 0.4 with GPU?

To reproduce go to [colab.research.google.com](url), choose a Python 3 runtime with GPU accelerator, and execute the following

0.4 code

```
%tensorflow_version 1.x

!pip install autokeras

import autokeras
import tensorflow

( ( x, y ), validation_data ) = tensorflow.keras.datasets.mnist.load_data( )
model = autokeras.ImageClassifier( verbose = True )
model.fit( x, y )
```

1.0 code

```
%tensorflow_version 2.x

!pip install git+git://github.com/keras-team/keras-tuner@master#egg=keras-tuner
!pip install git+git://github.com/keras-team/autokeras@master#egg=autokeras

import tensorflow
import autokeras

( ( x, y ), validation_data ) = tensorflow.keras.datasets.mnist.load_data( )
model = autokeras.ImageClassifier( )
model.fit( x, y, validation_data = validation_data )
```

### PR DESCRIPTION
**Pull request by jhfjhfj1**
### Which issue(s) does this Pull Request fix?

resolves #834 

### Details of the Pull Request

Updated image augmentation to improve the speed.
Originally, it tweaks all the images. Now, it only tweaks a portion of the images and has a probability for each operation.
The seed is removed because it uses the tensorflow seed.

### CODE REVIEW

**Comment by jhfjhfj1** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 6
           

Complexity increasing per file
==============================
- autokeras/tuner.py  7
- speed_test.py  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/autokeras/autokeras/pullRequest?prid=4551231&bid=15310345)

File: autokeras/hypermodel/preprocessor.py [hunk 1]
```diff
@@ -416,9 +415,10 @@ class ImageAugmentation(base.Preprocessor):
     """Collection of various image augmentation methods.
 
     # Arguments
+        percentage: Float. The percentage of data to augment.
         rotation_range: Int. The value can only be 0, 90, or 180.
             Degree range for random rotations. Default to 180.
-        random_crop: Boolean. Whether to crop the image randomly. Default to True.
+        crop: Boolean. Whether to crop the image randomly. Default to True.
```
Comment (fchollet) [inline autokeras/hypermodel/preprocessor.py]: Why change the name? `crop` could be interpreted in different ways: for instance, in the preprocessing layers API, we have `CenterCrop` and `RandomCrop`.
